include/system/nxlooper.h

```diff
@@ -161,6 +161,32 @@ void nxlooper_reference(FAR struct nxlooper_s *plooper);
 int nxlooper_setdevice(FAR struct nxlooper_s *plooper,
                        FAR const char *device);
 
+/****************************************************************************
+ * Name: nxlooper_loopraw
+ *
```

Comment:
```suggestion
 *
 * Description:
```

Reply:
All nxplayer.h/nxrecorder.h/nxlooper.h don't have this. It's better to fix the problem by new patch, do you think so?

Reply:
Ok, seems reasonable.